scripts/viz/feature_activation_plots.R

```diff
@@ -0,0 +1,132 @@
+# Pan-Cancer Variational Autoencoder
+# Gregory Way 2017
+#
+# Visualize VAE feature activation
+#
+# 1) Scatter plots of specific covariate information by encodings
+# 2) Heatmap of full encoded features by all samples
+
+library(readr)
+library(dplyr)
+library(ggplot2)
+
+# 1) Scatter plots of encoding specific sample activation
+# Load the combined data frame where columns are different data and algorithms
+vae_clinical_file <- file.path("data", "tybalt_features_with_clinical.tsv")
+combined_df <- readr::read_tsv(vae_clinical_file)
+
+# Load official colors
+tcga_colors <- readr::read_tsv(file.path("data", "tcga_colors.tsv"))
+
+tcga_colors <- tcga_colors[order(tcga_colors$`Study Abbreviation`), ]
+match_colors <- match(combined_df$acronym, tcga_colors$`Study Abbreviation`)
+combined_df$colors <- tcga_colors$`Hex Colors`[match_colors]
+
+palette_order <- c("BRCA", "PRAD", "TGCT", "KICH", "KIRP", "KIRC", "BLCA",
+                   "OV", "UCS", "CESC", "UCEC", "THCA", "PCPG", "ACC", "SKCM",
+                   "UVM", "HNSC", "SARC", "ESCA", "STAD", "COAD", "READ",
+                   "CHOL", "PAAD", "LIHC", "DLBC", "MESO", "LUSC", "LUAD",
+                   "GBM", "LGG", "LAML", "THYM", NA)
+
+plot_vae <- function(x, y, covariate, legend_show = TRUE) {
```

Comment:
Can you put this at the top of the script after the library calls? Also, `PlotVAE` would be more consistent with our current style guide.

Reply:
:+1: